tests/disaster-recovery/regional-dr/test_failover.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+from time import sleep
+
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import wait_for_nodes_status, get_node_objs
+
+logger = logging.getLogger(__name__)
+
+
+class TestFailover:
+    def test_failover_when_primary_site_up(self, rdr_workload):
+        """
+        Test Failover action when primary site is UP
+
+        """
+        logger.info("Waiting 10 minutes before initiating Failover")
+        sleep(600)
+        secondary_cluster_name = dr_helpers.get_secondary_cluster_name(
+            rdr_workload.namespace
+        )
+        dr_helpers.failover(secondary_cluster_name, rdr_workload.namespace)
```

Comment:
before failover are we checking for peerready="true" state ?

Reply:
Currently, we don't have that check in `failover()`
Also, this is not documented in official docs, do we still need to add that in ocs-ci?

Reply:
It is good to have this check added

Reply:
ack will add it as a part of `failover()` method